### PR TITLE
Clippy 1.51

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: askama_shared/src/generator.rs [hunk 1]
```diff
@@ -344,6 +344,8 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         buf.writeln(&format!("::askama_tide::try_into_body(&self, {:?})", &ext))?;
         buf.writeln("}")?;
         buf.writeln("}")?;
+
+        buf.writeln("#[allow(clippy::from_over_into)]")?;
```
Comment (vallentin) [inline askama_shared/src/generator.rs:348]: Did you check if it could be rewritten and `impl From` instead? or is it a false positive?

**Review by vallentin — APPROVED** [review]
Weird how I'm watching/subscribed to all activity, but I don't get a mail if the CI fails

File: askama_shared/src/generator.rs (hunk 1 above)
Comment (djc) [reply askama_shared/src/generator.rs:348]: I think it probably couldn't be done (`impl<T: Template> From<T> for tide::Response` would likely violate coherence?). In any case, seems to me that the conversion the other way around is pointless and doesn't make sense.

Comment (vallentin) [reply askama_shared/src/generator.rs:348]: Yeah, fair, was just curious :)

**Pull request merged**